server/README.md

```diff
@@ -123,6 +123,26 @@ More information can be found here https://rspec.info
 It is possible to set up an automated system so that when someone makes a pull request, or tries to merge to a branch, all tests are run automatically. A tool like Travis-CI can be used to do this. This has not been set up yet. 
 
 
+## Quote Style
+
+.JSON files use Double Quotes. 
+
+.rb files (ruby fies) use Single Quotes, except in the case that you would like to use string interpolation. 
+
+This is when you would like to sub some information into a string. 
```

Comment:
Sub is an abbreviated word, which a first-time reader may or may not catch on is "substitute".

Perhaps this sentence can use "substitute" instead?